src/jest/util.tsx

```diff
@@ -19,21 +20,19 @@ export const getMapMock = (override?: { [key: string]: any }) => ({
   getStyle: jest.fn(),
   getSource: jest.fn().mockReturnValue({ setData: jest.fn() }),
   project: jest.fn(),
-  ...override
+  getContainer: jest.fn(),
+  ...override,
 });
 
-export const renderWithMap = (comp: JSX.Element, mapValue: any) => {
-  return render(
-    <MapContext.Provider value={mapValue}>{comp}</MapContext.Provider>
-  );
+export const renderWithMap = (comp: ReactNode, mapValue: any) => {
+  return render(<MapContext.Provider value={mapValue}>{comp}</MapContext.Provider>);
 };
 
 export class MockComponent extends React.Component<{
   id: string;
-  map: MapboxGl.Map;
-  children?: JSX.Element | JSX.Element[];
+  map: TrimbleMaps.Map;
 }> {
-  public render() {
-    return <h1>{this.props.id}</h1>;
+  public render(): ReactNode {
+    return (<h1>{this.props.id}</h1>) as ReactNode;
```

Comment:
```suggestion
    return <h1>{this.props.id}</h1>;
```

This type assertion seems unnecessary.

Reply:
Good catch!